ui/app/pages/home/home.component.js

```diff
@@ -68,7 +93,36 @@ export default class Home extends PureComponent {
             query="(min-width: 576px)"
             render={() => <WalletView />}
           />
-          { !history.location.pathname.match(/^\/confirm-transaction/) ? <TransactionView /> : null }
+          { !history.location.pathname.match(/^\/confirm-transaction/)
+            ? (
+              <TransactionView>
+                {
+                  showPrivacyModeNotification && (
+                    <HomeNotification
```

Comment:
So if I read this right, this notification is only meant to be shown once, correct? And that is after the migration?

Reply:
Yes, once and only once